stock_secondary_unit/views/product_views.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2018 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_template_property_form" model="ir.ui.view">
+        <field name="name">Product template Secondary Unit</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="stock.view_template_property_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='inventory']" position="inside">
+                <group string="Secondary unit" groups="stock.group_stock_user">
+                    <field
+                        name="stock_secondary_uom_id"
+                        options="{'no_create': True}"
+                    />
+                </group>
+            </xpath>
+        </field>
+    </record>
+    <record id="product_template_form_view_procurement_button" model="ir.ui.view">
+        <field name="model">product.template</field>
+        <field
+            name="inherit_id"
+            ref="stock.product_template_form_view_procurement_button"
+        />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//button[@name='action_view_stock_move_lines']"
+                position="before"
+            >
+                <button
+                    type="object"
+                    name="action_update_quantity_on_hand"
+                    invisible="type != 'consu' and is_storable"
+                    class="oe_stat_button"
+                    icon="fa-building-o"
+                    groups="stock.group_stock_user"
+                >
+                    <div class="o_form_field o_stat_info">
+                        <span class="o_stat_value">
+                            <field
+                                name="secondary_unit_qty_available"
+                                widget="statinfo"
+                                nolabel="1"
+                            />
+                        </span>
+                        <span class="o_stat_text">
+                            <field name="stock_secondary_uom_id" readonly="1" />
+                        </span>
+                    </div>
+                </button>
+            </xpath>
+        </field>
+    </record>
+    <record id="product_form_view_procurement_button" model="ir.ui.view">
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="stock.product_form_view_procurement_button" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//button[@name='action_update_quantity_on_hand']"
+                position="after"
+            >
+                <button
+                    type="object"
+                    name="action_update_quantity_on_hand"
+                    invisible="type != 'consu' and is_storable"
```

Comment:
Same as above
`                    invisible="type != 'consu' and not is_storable"
`